undeploy.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"errors"
+	"os"
+	"path"
+	"runtime"
+	"strings"
+
+	"github.com/portainer/docker-compose-wrapper/compose"
+)
+
+var errUndeployComposeFailure = errors.New("compose stack remove failure")
+
+func (cmd *UndeployCommand) Run(cmdCtx *CommandExecutionContext) error {
+	cmdCtx.logger.Infow("Undeploying Compose stack from Git repository",
```

Comment:
What's an undeployment?

Reply:
there is a file called "undeploy.go" to handle undeployment.

Reply:
Opposite of deploy?